locale/ru-RU.json

```diff
@@ -71,77 +71,77 @@
     "status": {
       "name": "Статус",
       "online": "Онлайн",
-      "offline": "Не в сети"
+      "offline": "Офлайн"
     },
     "address": "Адрес",
-    "players": "Игроки",
-    "pastebin": "Pastebin",
-    "distance": "Расстояние",
-    "statistics": "Статистика сервера",
-    "daylight-saving-time": "Переход на летнее время не влияет на время на графике.",
+    "players": "Игроков",
+    "pastebin": "Инфо",
+    "distance": "Расстояние до вас",
+    "statistics": "Статистика",
+    "daylight-saving-time": "Переход на летнее время не влияет на график.",
     "graph": {
-      "loading": "Загрузка данных с сервера...",
-      "visitors": "Посетители",
+      "loading": "Загрузка данных...",
+      "visitors": "Игроков",
       "visitor-suffix": "",
       "options": {
         "date-format": "yyyy-MM-dd aa hh:mm",
-        "invalid-start-time": "The start time must be earlier than the stop time.",
-        "invalid-stop-time": "The stop time must be later than the start time.",
-        "data-range": "Data Period",
-        "start-time": "Start Time",
-        "stop-time": "Stop Time",
-        "data-resolution": "Data Resolution",
-        "graph-update-btn": "Update Graph",
+        "invalid-start-time": "Начало должно быть раньше конца.",
+        "invalid-stop-time": "Конец должен быть позже начала.",
+        "data-range": "Период",
+        "start-time": "Начало",
+        "stop-time": "Конец",
+        "data-resolution": "Масштаб",
+        "graph-update-btn": "Обновить график",
         "period": {
-          "custom": "Custom",
-          "1hour": "1 Hour",
-          "8hour": "8 Hours",
-          "1day": "1 Day",
-          "2day": "2 Days",
-          "1week": "1 Week",
-          "1month": "1 Month",
-          "3month": "3 Months",
-          "1year": "1 Year",
-          "4year": "4 Years"
+          "custom": "Свой",
+          "1hour": "Час",
+          "8hour": "8 Часов",
+          "1day": "Сутки",
+          "2day": "2 Суток",
+          "1week": "Неделя",
+          "1month": "Месяц",
+          "3month": "Квартал",
+          "1year": "Год",
+          "4year": "4 Года"
         },
         "resolution": {
-          "1m": "1 Minute",
-          "5m": "5 Minutes",
-          "1h": "1 Hour",
-          "1d": "1 Day"
+          "1m": "1 Минута",
+          "5m": "5 Минут",
+          "1h": "Час",
+          "1d": "День"
         }
       }
     }
   },
   "footer": {
-    "notice": "SCP: Secret Laboratory - игра, выпущенная Northwood Studios ™, этот сайт не имеет отношения к Northwood Studios.™.<br/>Все товарные знаки и зарегистрированные товарные знаки являются собственностью соответствующих владельцев."
+    "notice": "SCP: Secret Laboratory - игра, выпущенная Northwood Studios™. Сайт не имеет никакого отношения к студии Northwood.<br/>Все товарные знаки и зарегистрированные товарные знаки являются собственностью соответствующих владельцев."
```

Comment:
Please amend the paragraph `Northwood Studios.™` to keep the original translation.